src/index.js

```diff
@@ -80,14 +80,15 @@ function clickLog(event) {
 }
 
 function validateForm() {
-    let validated = false;
-    let selectedDate = document.querySelector('.input-date');
-    let selectedDuration = document.querySelector('.input-duration');
-    let selectedTravelers = document.querySelector('.input-travelers');
-    let selectedDestination = document.querySelector('.input-destination');
-    let dateError = document.querySelector('.date-error');
-    let durationError = document.querySelector('.duration-error');
-    let travelersError = document.querySelector('.travelers-error');
+    let validated = true;
+    // let validated = false;
+    const selectedDate = document.querySelector('.input-date');
+    const selectedDuration = document.querySelector('.input-duration');
+    const selectedTravelers = document.querySelector('.input-travelers');
+    const selectedDestination = document.querySelector('.input-destination');
+    const dateError = document.querySelector('.date-error');
+    const durationError = document.querySelector('.duration-error');
+    const travelersError = document.querySelector('.travelers-error');
     // let destinationError = document.querySelector('.destination-error');
```

Comment:
Good catch changing the `let`s to `const`s

Reply:
"I don't know what that means but i like it!!" - Ricky Bobby's Kid